ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/srma/editSrmaPage.ts

```diff
@@ -361,6 +361,24 @@ export class EditSrmaPage {
         return this;
     }
 
+    public verifyTextHint()
```

Comment:
hasReasonHintText would describe what it is looking for

If we had another hint text to check for it would get confusing